Library/Formula/ctags.rb

```diff
@@ -2,6 +2,7 @@ class Ctags < Formula
   homepage "http://ctags.sourceforge.net/"
   url "https://downloads.sourceforge.net/ctags/ctags-5.8.tar.gz"
   sha1 "482da1ecd182ab39bbdc09f2f02c9fba8cd20030"
+  version "5.8-p1"
```

Comment:
We can't make up our own versions here. We could add a `revision` if it's important enough to justify making everyone upgrade or just leave it as-is and update the bottle.